apps/saleor-app-checkout/Dockerfile

```diff
@@ -0,0 +1,48 @@
+# Export package.json's in monorepo using turborepo
+# https://turborepo.org/docs/handbook/deploying-with-docker#the-solution
+FROM node:16-slim AS builder
+WORKDIR /app
+
+ARG PNPM_VERSION=7.13.4
```

Comment:
Why are we using different pnpm version than in Dockerfile.base?

Reply:
That's a leftover from trying to build checkout in standalone mode - I've removed it